Gruntfile.js

```diff
@@ -150,6 +150,33 @@ module.exports = function( grunt ) {
 					build_dir: 'build'
 				}
 			}
+		},
+
+		// Generate POT files.
+		makepot: {
+			target: {
+				options: {
+					potFilename: '<%= pkg.name %>.pot',
+					exclude: [
+						'dev-lib/.*',
+						'node_modules/.*',
+						'tests/.*',
+						'vendor/.*'
+					],
+					processPot: function( pot ) {
+						pot.headers['project-id-version'];
+						return pot;
+					},
+					type: 'wp-plugin',
+					domainPath: 'languages',
+					potHeaders: {
+						poedit: true,
+						'x-poedit-keywordslist': true,
+						'last-translator': '<%= pkg.author %>',
+						'language-team': '<%= pkg.author %>'
+					}
+				}
+			}
```

Comment:
No, it was not. The PR has grown beyond the initial scope and I need to rename it. This actually should have been in the repo from back when I added Grunt, but was accidentally deleted when I removed the connection with transifex.